deeplearning4j-nlp-parent/deeplearning4j-nlp-chinese/src/main/java/org/apdplat/word/WordFrequencyStatistics.java

```diff
@@ -0,0 +1,288 @@
+/**
+ *
+ * APDPlat - Application Product Development Platform
+ * Copyright (c) 2013, 杨尚川, yang-shangchuan@qq.com
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.apdplat.word;
```

Comment:
This thing states GPL in header. Isn't that a blocker? cc @agibsonccc @huitseeker